templates/repo/issue/labels/label_edit_modal.tmpl

```diff
@@ -50,7 +50,7 @@
 			<div class="field">
 				<label for="color">{{ctx.Locale.Tr "repo.issues.label_color"}}</label>
 				<div class="column js-color-picker-input">
-					<input name="color" value="#70c24a" placeholder="#c320f6" required maxlength="7">
+					<input name="color" value="#70c24a" placeholder="#c320f6" required pattern="^#?([\dA-Fa-f]{3}|[\dA-Fa-f]{6})$" maxlength="7">
```

Comment:
```suggestion
					<input name="color" value="#70c24a" placeholder="#c320f6" required pattern="^#?[0-9a-fA-F]{3,6}$" maxlength="7">
```

same as above with optional `#` kept.

Reply:
No, that's NOT  right.

NOT RIGHT

I don't want to waste any time on this PR.

![image](https://github.com/user-attachments/assets/d079f36e-7c5c-432c-864c-7f7ac242f154)

Reply:
Ah you are right `{3,6}` matches between 3 and 6 occurances, not exactly.